Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
 
           def docker_base = "px4io/px4-dev-base:2018-03-30"
           def docker_nuttx = "px4io/px4-dev-nuttx:2018-03-30"
-          def docker_ros = "px4io/px4-dev-ros:2018-03-30"
+          def docker_ros = "px4io/px4-dev-ros:2018-06-19"
```

Comment:
Why is this being changed as part of this PR? Aren't the changes to the container only needed for your odometry msg PR?